emcache/cluster.py

```diff
@@ -221,6 +254,80 @@ def node(self, memcached_host_address: MemcachedHostAddress) -> Node:
 
         raise ValueError(f"{memcached_host_address} can not be found")
 
+    async def _autodiscovery_monitor(self) -> None:
+        logger.debug("Autodiscovery task started")
+        while True:
+            if not self._autodiscovery:
```

Comment:
Is this really necessary? so if `_autodiscovery_monitor` was executed was because `self._autodiscovery` was configured as true, right? if so seems to me that this is not really necessary.

Reply:
You're right, I will remove it.